modules/bridge/src/main/ts/ephox/bridge/components/menu/Title.ts

```diff
@@ -0,0 +1,28 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
+import { Optional, Result } from '@ephox/katamari';
+
+export interface TitleSpec {
+  type: 'title';
+  text: string;
+  name?: string;
+  classes?: string[];
```

Comment:
Another place where we shouldn't be including classes.

Reply:
See https://github.com/tinymce/tinymce/pull/6149#discussion_r515868104